docs/src/appendix/cheatcodes/start_mock_call.md

```diff
@@ -2,17 +2,28 @@
 
 > `fn start_mock_call<T, impl TSerde: serde::Serde<T>, impl TDestruct: Destruct<T>>(contract_address: ContractAddress, function_name: felt252, ret_data: T)`
 
-Mocks contract call to a `function_name` of a contract at the given address. A call to function `function_name` will return data provided in `ret_data` argument. 
+Mocks contract call to a `function_name` of a contract at the given address. A call to function `function_name` will
+return data provided in `ret_data` argument.
 
-If there is a contract deployed at the given address, mocked function won't be executed. Address with no contract can be mocked as well.
+If there is a contract deployed at the given address, mocked function won't be executed. Address with no contract can be
+mocked as well.
 Mock can be canceled with [`stop_mock_call`](./stop_mock_call.md).
 
 - `contract_address` - target contract address
 - `function_name` - name of the function in a contract at the `contract_address` that will be mocked
 - `ret_data` - data to return by the function `function_name`
 
 > 📝 **Note**
-> The inner call (i.e. when a contract calls a function from within itself) cannot be mocked.
+> Mocks do not have any effect on function calls withing the contract itself.
+> E.g. for a function withing a contract defined like this:
+>
+> ```rust
+> fn function_a(self: @ContractState) {
+>   function_b()
+> } 
+> ```
+>
+> Mocking `function_b` would have no effect.
```

Comment:
```suggestion
> mocking `function_b` would have no effect when calling `function_a`.
```

Reply:
It's below a snipped and we have `:` at the end of the previous sentence so I think it should start with capital letter?